app/src/main/res/values-de/strings.xml

```diff
@@ -15,7 +15,7 @@
     <string name="permission_explanation">Der Emulator benötigt Schreibrechte für den externen Speicher
                 des Geräts um TRS-80 ROM- und Disketten-Images zu speichern. Der Emulator funktioniert ohne diese
                 Schreibrechte nicht.</string>
-    <string name="permission_request">Schreibrechte anfragen</string>
+    <string name="permission_request">OK ich verstehe</string>
```

Comment:
Sorry, buddy. As someone who sports a German last name, you need to take responsibility! ;)